src/Protocol/Protocol.php

```diff
@@ -212,15 +212,19 @@ public function getAckFrame(Frame $frame, $transactionId = null)
      *
      * @param \Stomp\Transport\Frame $frame
      * @param string $transactionId
+     * @param bool $requeue Requeue header supported on RabbitMQ >= 3.4
```

Comment:
shouldn't we implement it in the RabbitMq class then?

Reply:
that would probably make more sense. should the base protocol then also throw the LogicException if $requeue !== null?

Reply:
yes, I think so.